readme.md

```diff
@@ -128,6 +128,11 @@ Default: `false`
 
 Reveal the downloaded file in the system file manager, and if possible, select the file.
 
+#### showBadge
+
+Type: `boolean`<br>
+Default: `false`
+
```

Comment:
Needs a short description. It's for example not clear we're talking about the badge on the dock icon.